api/lib/infrastructure/repositories/campaign-management-repository.js

```diff
@@ -2,10 +2,14 @@ const _ = require('lodash');
 const { knex } = require('../bookshelf');
 const CampaignManagement = require('../../domain/read-models/CampaignManagement');
 const { fetchPage } = require('../utils/knex-utils');
+const CampaignParticipation = require('../../domain/models/CampaignParticipation');
+const Campaign = require('../../domain/models/Campaign');
+
+const { SHARED, TO_SHARE, STARTED } = CampaignParticipation.statuses;
```

Comment:
Ici même suggestion qu'en haut
```suggestions
const CampaignParticipationStatuses = require('../../../../lib/domain/models/CampaignParticipationStatuses');
const { STARTED, SHARED, TO_SHARE } = CampaignParticipationStatuses;
```

Reply:
yes